### PR TITLE
Update README, add API section

### PR DESCRIPTION
**Pull request by Vijar**
@redonkulus @mridgway

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: README.md [hunk 1]
```diff
@@ -32,6 +32,29 @@ app.createContext({
 });
 ```
 
+### Registering Your Services
+
+Since the fetchr plugin is in control the fetchr class, we expose this through the `registerService` method.
```
Comment (jedireza) [inline README.md]: Add backticks to fetchr => `fetchr`?

**Comment by Vijar** [comment]
Not sure if those relative links will work or not

**Comment by jedireza** [comment]
:boat:

**Comment by redonkulus** [comment]
:shipit:

File: README.md [hunk 2]
```diff
@@ -45,19 +68,21 @@ app.plug(fetchrPlugin({
...
 
-### actionContext
+### Instance Methods
 
- * `actionContext.service.read(resource, params, [config,] callback)`: Call the read method of a service. See [fetchr docs](https://github.com/yahoo/fetchr) for more information.
- * `actionContext.service.create(resource, params, body, [config,] callback)`: Call the create method of a service. See [fetchr docs](https://github.com/yahoo/fetchr) for more information.
- * `actionContext.service.update(resource, params, body, [config,] callback)`: Call the update method of a service. See [fetchr docs](https://github.com/yahoo/fetchr) for more information.
- * `actionContext.service.delete(resource, params, [config,] callback)`: Call the delete method of a service. See [fetchr docs](https://github.com/yahoo/fetchr) for more information.
- * `actionContext.getServiceMeta()`: The plugin will collect metadata for service responses and provide access to it via this method. This will return an array of metadata objects.
+#### Constructor(options)
```
Comment (Vijar) [inline README.md]: Not instance method. Move out.

File: README.md [hunk 3]
```diff
@@ -45,19 +68,21 @@ app.plug(fetchrPlugin({
...
+
+Creates a new fetchr plugin instance with the following parameters:
 
-## Other Methods
+ * `options`: An object containing the plugin settings
+ * `options.xhrPath` (optional): Stores your xhr path prefix used by client side requests
```
Comment (Vijar) [inline README.md]: document DEFAULT value

**Pull request merged**